### ISSUE TITLE
TypeError: SourceMapConsumer.with is not a function

### ISSUE THREAD

**Issue by witschas** [fearnycompknowhow/vue-typescript-debugger#1]
Hi,

thanks for your blogpost here https://ckhconsulting.com/vue-typescript-debuggers/
I had really success with all of your steps, except the last one.
I'm working on a ionic vue project and get an error:

TypeError: SourceMapConsumer.with is not a function

when running 
`ionic serve`
(which is nothing else than 
`vue-cli-service serve` 

Do you have an idea what to change or to do?

here is the complete error feedback
`$ ionic serve
> vue-cli-service serve
[vue-cli-service]  INFO  Starting development server...
[vue-cli-service]  ERROR  TypeError: SourceMapConsumer.with is not a function
[vue-cli-service] TypeError: SourceMapConsumer.with is not a function
[vue-cli-service]     at /Users/rico/_repos/99v/vantcode-clubnews-vue-app/vue.config.js:67:74
[vue-cli-service]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:28:17)
[vue-cli-service]     at AsyncSeriesHook.lazyCompileHook (/Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/tapable/lib/Hook.js:154:20)
[vue-cli-service]     at Compilation.finish (/Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/webpack/lib/Compilation.js:1253:28)
[vue-cli-service]     at /Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/webpack/lib/Compiler.js:672:17
[vue-cli-service]     at _done (eval at create (/Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[vue-cli-service]     at eval (eval at create (/Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:32:22)
[vue-cli-service]     at /Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/webpack/lib/Compilation.js:1185:12
[vue-cli-service]     at /Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/webpack/lib/Compilation.js:1097:9
[vue-cli-service]     at processTicksAndRejections (node:internal/process/task_queues:76:11)
[vue-cli-service] node:internal/process/promises:227
[vue-cli-service]           triggerUncaughtException(err, true /* fromPromise */);
[vue-cli-service]           ^
[vue-cli-service] RpcIpcMessagePortClosedError: Cannot send the message - the message port has been closed for the process 52135.
[vue-cli-service]     at /Users/rico/_repos/99v/vantcode-clubnews-vue-app/node_modules/@vue/cli-plugin-typescript/node_modules/fork-ts-checker-webpack-plugin-v5/lib/rpc/rpc-ipc/RpcIpcMessagePort.js:47:47
[vue-cli-service]     at processTicksAndRejections (node:internal/process/task_queues:80:21) {
[vue-cli-service]   code: undefined,
[vue-cli-service]   signal: undefined
[vue-cli-service] }

[ERROR] vue-cli-service has unexpectedly closed (exit code 1).`

**Comment by fearnycompknowhow** [comment]
Hi @99cv, I'm glad you found my blog useful!

Are you encountering the problem on this repository, or is it happening within your own personal project?  It seems to be the latter, but I would just like to confirm.

Also, what version of the `source-map` package are you using?  If you take a look at the `package.json` file within this repo, you'll notice that this project is using version `^0.7.3`.

Lastly, is your project using Vue 2 or Vue 3?

**Comment by witschas** [comment]
Hi @fearnycompknowhow 

yes, your blogpost was very useful, thanks again.
First things first:
- yes, I implemented this on my own project
- and, it's a Vue3 project.

The version of source-map was a very important hint. I added source-map via 

`npm i --save-dev source-map`

but there was an older version installed (^0.5.7) than yours. So I manually bumped the version up to your version and the error went away, yeeha!

I had another issue then with the variable original in line 74 of vue.config.js (it was never initiated, so I added a line 

`let original = null;`

in line 68 and it worked out well after that change.

So thanks for your response and helping me out.
